OWNERS

```diff
@@ -17,5 +17,82 @@ emeritus_approvers:
 - stewart-yu
 # - zacharysarah, commented out to disable PR assignments
 
+filters:
```

Comment:
_Optional suggestion:_ It would also be helpful to include the appropriate configuration that adds correct language label _(language/<2-letter code>)_ for changes to localized README.

Reply:
Thanks for your review!

Sounds good, fixed by https://github.com/kubernetes/website/pull/47763/commits/fe4ffa18ea89cedc723ed91774598ec940921110. Thanks!